tftp/util.py

```diff
@@ -41,8 +44,7 @@ def iterate(_=None):
             else:
                 return deferLater(clock, delay, call).addCallback(iterate)
         else:
-            # No timings were given.
-            return succeed(None)
+            raise ValueError("No timings specified.")
```

Comment:
Ah, I suppose, the idea is that timings is an _iterable_, so you can't _trivially_ check it for emptiness.